FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -374,7 +371,7 @@ - (void)fetchWithUserProperties:(NSDictionary *)userProperties
 
       if (error || (statusCode != kRCNFetchResponseHTTPStatusCodeOK)) {
         // Update metadata about fetch failure.
-        [strongSelf->_settings updateMetadataWithFetchSuccessStatus:NO];
+        [strongSelf->_settings updateMetadataWithFetchSuccessStatus:NO templateVersion:@"1"];
```

Comment:
it looks like templateVersion isn't updated if fetch fails, but i would pass templateVersion = 0 or nil just to be clear.

Reply:
Done.